src/brackets.js

```diff
@@ -142,7 +142,7 @@ define(function (require, exports, module) {
     
     // read URL params
     params.parse();
-    
+
```

Comment:
@busykai There was some piece of code here in the earlier commits and i just deleted that piece of code based on the code review comments and that is why it is showing as `blank space removal` change. I could revert this, but I think it is a good idea to remove the extra spaces wherever possible.